draft-ietf-gnap-core-protocol.md

```diff
@@ -2690,6 +2692,14 @@ request and MUST be protected by HTTPS or equivalent means.
 When the end user is directed to enter a short code through the ["user_code"](#response-interact-usercode)
 mode, the client instance communicates the user code to the end user and
 directs the end user to enter that code at an associated URI.
+The client instance MAY
```

Comment:
Not sure about this memorability thing, or at least I believe it can remain something unspecified (potentially allowed by some implementations if they wish). It's not so easy to make sure this doesn't open new routes of attack. Or else we'd need to explain how to handle the fuzziness.

Reply:
I think it's OK here because the fuzziness is only in formatting and only if the resulting format is equivalent to the actual value that needs to be input by the user. This is basically acknowledging something that people already do in practice, but putting the right guides around it in the hopes that things are done correctly.

Reply:
OK